wordpress-mobile/WordPress-iOS
# Improve voice over support for reader's post details view

Implements #8712 

To test:
1. Navigate to Reader
2. Start tapping posts until reaching the details view
3. Activate voice over

Expected behaviour:

1. The post title should be readable by VoiceOver, and provide blog title, post author and post date
2. The ellipsis button should identify as a button and provide a hint
3. The post content is exposed to VoiceOver, with UITextView's default behaviour
